pkg/cli/context.go

```diff
@@ -105,7 +105,6 @@ func initCLIDefaults() {
 	debugCtx.replicated = false
 	debugCtx.inputFile = ""
 	debugCtx.printSystemConfig = false
-	debugCtx.maxResults = 1000
```

Comment:
Keep this line; it's needed. You can change the value to zero.

Reply:
done